src/Compilers/VisualBasic/Test/Semantic/Diagnostics/OperationAnalyzerTests.vb

```diff
@@ -1191,6 +1191,15 @@ Class C
         M0(1, New Integer() { 2, 3, 4 })
         M0(1, New Integer() { 2, 3, 4, 5 })
         M0(1, New Integer() { 2, 3, 4, 5, 6 })
+        Dim local As D = new D(1, 2, 3, 4, 5)
```

Comment:
Yes. VB prohibits writing the declaration of a method or constructor with a ParamArray after an Optional parameter, so constructing the test is difficult. The way the VB compiler processes argument lists prevents the issue from arising in VB anyway, so the specific issue cannot arise.